### PR TITLE
duplicated 'the' and problem of "a" and "an"

### PR DESCRIPTION
**Pull request by SRaddict**

### CODE REVIEW

**Comment by SRaddict** [comment]
added another error：fix a series errors of using "a" and "an"

File: docs/admin/cluster-large.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-title: Building Large Clusters
----
-
+---
+assignees:
+- davidopp
+- lavalamp
+title: Building Large Clusters
+---
+
```
Comment (xulike666) [inline docs/admin/cluster-large.md]: Do not change the default markdown format.

File: docs/getting-started-guides/mesos/index.md [hunk 2]
```diff
@@ -229,7 +229,7 @@ We assume that kube-dns will use
 
 Note that we have passed these two values already as parameter to the apiserver above.
 
-A template for an replication controller spinning up the pod with the 3 containers can be found at [cluster/addons/dns/skydns-rc.yaml.in][11] in the repository. The following steps are necessary in order to get a valid replication controller yaml file:
+A template for a replication controller spinning up the pod with the 3 containers can be found at [cluster/addons/dns/skydns-rc.yaml.in][11] in the repository. The following steps are necessary in order to get a valid replication controller yaml file:
```
Comment (xulike666) [inline docs/getting-started-guides/mesos/index.md]: There is no wrong with "an replication" and please revert it.

File: docs/user-guide/connecting-applications.md [hunk 3]
```diff
@@ -181,7 +181,7 @@ default-token-il9rc   kubernetes.io/service-account-token   1
 nginxsecret           Opaque                                2
 ```
 
-Now modify your nginx replicas to start a https server using the certificate in the secret, and the Service, to expose both ports (80 and 443):
+Now modify your nginx replicas to start an https server using the certificate in the secret, and the Service, to expose both ports (80 and 443):
```
Comment (xulike666) [inline docs/user-guide/connecting-applications.md]: There is no wrong with "a https" and revert this.

**Review by xulike666 — CHANGES_REQUESTED** [review]
Please sure you use the "a/an" in the right way.

**Review by xulike666 — APPROVED** [review]

**Pull request merged**